contracts/test/PDPService.t.sol

```diff
@@ -338,31 +366,96 @@ contract PDPServiceProofSetMutateTest is Test {
         assertEq(pdpService.getRootLeafCount(setId, 2), 0);
 
     }
-
-    function testRemoveBadRootDoesntFail() public {
+    
+    function testRemoveFutureRoots() public {
         uint256 setId = pdpService.createProofSet(address(recordKeeper));
         recordAssert.expectRecord(PDPListener.OperationType.CREATE, setId);
         PDPService.RootData[] memory roots = new PDPService.RootData[](1);
         roots[0] = PDPService.RootData(Cids.Cid(abi.encodePacked("test")), 64);
         pdpService.addRoots(setId, roots);
         recordAssert.expectRecord(PDPListener.OperationType.ADD, setId);
-        uint256[] memory toRemove = new uint256[](1);
-        toRemove[0] = 1;
-        pdpService.removeRoots(setId, toRemove);
+        assertEq(true, pdpService.rootChallengable(setId, 0));
```

Comment:
Assert root 1 is not challengable

Reply:
Good point, but it makes more sense to assert its not live as that's the condition that causes failure on removal schedule.